meson.build

```diff
@@ -33,8 +40,11 @@ int main(int argc, char **argv)
 if not cxx.links(code, dependencies: deps, name: 'libicecc links dynamically')
     icecc = dependency('icecc', static: true)
     liblzo2 = cxx.find_library('lzo2')
-    libcapng = dependency('libcap-ng')
-    deps += [libcapng, liblzo2]
+    deps += [liblzo2]
+    if build_machine.system() == 'linux'
+        libcapng = dependency('libcap-ng')
+        deps += [liblzo2]
```

Comment:
Likewise:

```
libcapng = depedency('libcap-ng', required: false)
if libcapng.found()
    deps += [libcapng]
endif

Reply:
Apparently `meson` can actually deal with optional libraries in the `deps` list even if they are not found. Hence, the `if libcapng.found()` doesn't seem to be necessary.